### PR TITLE
Docs: Reviewing a Pull request

### PR DESCRIPTION
**Pull request by mark-pkgflow**
## Summary
This update introduces a comprehensive guide aimed at empowering non-developer team members to actively participate in the PR review process. The guide covers essential aspects such as suggesting changes on specific lines, general commenting, verifying links, and assessing mobile responsiveness.

## Changes
- Added detailed instructions on how to suggest changes to specific lines in a PR, enhancing precision in feedback.
- Included guidelines for leaving both line-specific and general comments, facilitating clearer communication.
- Introduced a step-by-step process for checking the functionality of links within the PR, ensuring all resources are accessible and correct.
- Updated the mobile responsiveness section to incorporate the use of the QR code provided by the Netlify bot, allowing for straightforward mobile testing.

## Rationale
The ability for non-developers to effectively review PRs is crucial for maintaining high-quality content and functionality across our projects. This update aims to demystify the PR review process for non-devs, fostering a more inclusive and collaborative environment.

## Testing
- Please review the added sections for clarity and comprehensiveness.
- Suggestions for further simplification or any areas that might require additional detail are welcome.

## Preview
To preview the changes click [here](https://github.com/DFE-Digital/improve-workload-and-wellbeing-for-school-staff/blob/25818ac31bd6527228bcc517ef0558e93695d72d/docs/reviewing-pull-request.md).

### CODE REVIEW

**Comment by mark-pkgflow** [comment]
[Add guidance on how to best review pull requests to our guidance on adding resources](https://trello.com/c/n7znx1x7/369-add-guidance-on-how-to-best-review-pull-requests-to-our-guidance-on-adding-resources)

**Review by AbigailMcP — APPROVED** [review]
Love it!

**Comment by claire-hughez** [comment]
This looks perfect to me :)

When 'reviewing links', I've made 1 suggestion

File: docs/reviewing-pull-request.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+2. **Leave a comment**:
+   - For general comments, use the main comment box at the bottom of the `Conversation` tab.
+   - For file-specific comments without suggesting changes, use the comment box as described in the previous section, but without adding a suggestion block.
+3. **Submit your comment**: Click `Comment` to post it.
+
+## Checking Links in the PR
+
+1. **Access the deployed preview**: Look for the Netlify bot comment in the `Conversation` tab of the PR. It should provide a link to the deployed preview of the changes.
+2. **Test the links**: Click through all the links in the deployed preview to ensure they work correctly. Pay special attention to new resources or modified links.
+
```
Comment (claire-hughez) [inline docs/reviewing-pull-request.md:28]:
```suggestion
3. **Check resources are linked correctly**: Make sure any new resources are linked to both on the staff wellbeing page or the workload reduction toolkit **and** the appropriate explore resources section.
```

**Pull request merged**